src/cli/README_TCAT.md

```diff
@@ -2,16 +2,103 @@
 
 ## Command List
 
+- advid [#advid]
+- devid [#devid]
 - help [#help]
 - start [#start]
 - stop [#stop]
 
+### advid
+
+Displays currently set TCAT advertised ids.
+
+```bash
+tcat advid
+type oui24, value: f378aa
+Done
+```
+
+### advid ianapen \<id\>
+
+Sets TCAT advertised ianapen id.
+
+```bash
+tcat advid ianapen f378aabb
+Done
+```
+
+### advid oui24 \<id\>
+
+Sets TCAT advertised oui24 id.
+
+```bash
+tcat advid oui24 f378aa
+Done
+```
+
+### advid oui36 \<id\>
+
+Sets TCAT advertised oui36 id.
+
+```bash
+tcat advid oui36 f378aabbcc
+Done
+```
+
+### advid discriminator \<id\>
+
+Sets TCAT advertised discriminator id.
+
+```bash
+tcat advid discriminator f378aabbdd
+Done
+```
+
+### advid empty
+
+Clears TCAT advertised id.
+
+```bash
+tcat advid empty
+Done
+```
+
+### devid
+
+Displays currently set TCAT device id.
+
+```bash
+tcat devid
+abcd
+Done
+```
+
+### devid \<id\>
+
+Sets TCAT device id.
+
+```bash
+tcat devid abcd
+Done
+```
+
+### devid empty
+
+Clears TCAT device id.
+
+```bash
+tcat devid empty
```

Comment:
in OT CLI commands, always `clear` is used to clear an item, never `empty`. Change to `clear` ?